### PR TITLE
Cloud9 integration for Session_manager

### PR DESCRIPTION
**Pull request by snjkumar23**
## What does this PR do and why?

Cloud9 Changes for Session Manager section

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/basics/operations/session-manager/index.md [hunk 1]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
 
 Go to the AWS console and update your stack with a new template.
 
+:::::tabs{variant="container"}
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace`:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  cd cfn101-workshop/code/workspace
+  :::
+2. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you.
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md]:
```suggestion
2. Use the AWS CLI to create the stack. The required parameter `--template-body` have been pre-filled for you.
```

File: content/basics/operations/session-manager/index.md [hunk 2]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
+  :::code{language=shell showLineNumbers=false showCopyAction=false}
+  aws cloudformation update-stack --stack-name cfn-workshop-resources --template-body file://session-manager.yaml --parameters ParameterKey="EnvironmentType",ParameterValue="Dev" --capabilities CAPABILITY_NAMED_IAM
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md]:
```suggestion
  aws cloudformation create-stack --stack-name cfn-workshop-session-manager --template-body file://session-manager.yaml --parameters ParameterKey="EnvironmentType",ParameterValue="Dev" --capabilities CAPABILITY_NAMED_IAM
```

File: content/basics/operations/session-manager/index.md [hunk 3]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
+  :::
+  :::alert{type="info"}
+  For System Manager to work, the instance need to meet following conditions:
+
+    \- **Access to the internet, or a VPC Endpoint.** \
+    \- **Role attached with correct permission.** \
+  By changing the environment, instance will be stopped and started again. This will help to start `ssm-agent` which may have timed-out as the role wasn't attached in a previous lab.
+  :::
+3. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md]:
```suggestion
3. If the `create-stack` command was successfully sent, CloudFormation will return `StackId`.
```

File: content/basics/operations/session-manager/index.md [hunk 4]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
+  :::code{language=shell showLineNumbers=false showCopyAction=false}
+  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-resources/739fafa0-e4d7-11ed-a000-12d9009553ff"
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md]:
```suggestion
  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-session-manager/739fafa0-e4d7-11ed-a000-12d9009553ff"
```

File: content/basics/operations/session-manager/index.md [hunk 5]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
+  :::
+4. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **UPDATE_COMPLETE**.
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md]:
```suggestion
4. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **CREATE_COMPLETE**.
```

File: content/basics/operations/session-manager/index.md [hunk 6]
```diff
@@ -102,29 +102,56 @@ WebServerInstance:
...
 :::alert{type="info"}
 For System Manager to work, the instance need to meet following conditions:
   \- **Access to the internet, or a VPC Endpoint.** \
   \- **Role attached with correct permission.** \
 By changing the environment, instance will be stopped and started again. This will help to start `ssm-agent` which may have timed-out as the role wasn't attached in a previous lab.
 :::
-
-1. You can leave **Configure stack options** default, click **Next**.
+1. Choose to accept default values for **Configure stack options**; choose **Next**.
 1. On the **Review <stack_name>** page, scroll down to the bottom and tick **I acknowledge that AWS CloudFormation might create IAM resources** check box, then click on **Update stack**.
```
Comment (rezabekf) [inline content/basics/operations/session-manager/index.md:150]:
```suggestion
1. On the **Review <stack_name>** page, scroll down to the bottom and tick **I acknowledge that AWS CloudFormation might create IAM resources** check box, then click on **Create stack**.
```

**Review by rezabekf — CHANGES_REQUESTED** [review]
Updated local development.

**Comment by snjkumar23** [comment]
incorporated changes

**Comment by snjkumar23** [comment]
> Updated local development.

**Pull request closed without merging**